lib/minimart/version.rb

```diff
@@ -1,3 +1,3 @@
 module Minimart
-  VERSION = "1.0.2"
+  VERSION = "1.0.3"
```

Comment:
is this a significant enough feature to go up to 1.1.0 ?  see http://semver.org/ ... if he doesn't break backward compatibility then i think we should bump the middle number